src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,129 @@
+package duke;
+
+import java.util.ArrayList;
+
+import duke.task.Task;
+
+/**
+ * Ui is a class that prints messages to the user.
+ */
+public class Ui {
+    /**
+     * Prints a welcome message.
+     * @return A string containing a welcome message.
+     */
+    public static String printWelcomeMessage() {
+        String introduction = "Hello! I'm NEKOBOT!!\n";
+        String question = "What can I do for you :>";
+        return introduction + question;
+    }
+
+    /**
+     * Prints a goodbye message.
+     * @return A string containing a goodbye message.
+     */
+    public static String printGoodbyeMessage() {
+        return "Bye~ Hope to see you again soon ;w;";
+    }
+
+    /**
+     * Prints all tasks in the TaskList.
+     * @param taskList An ArrayList containing all existing Tasks.
+     * @return A string containing all tasks in the TaskList.
+     */
+    public static String printListMessage(ArrayList<Task> taskList) {
+        String preface = "Here are the tasks in your list!!\n";
+        StringBuilder tasks = new StringBuilder();
```

Comment:
unintuitive variable name for a stringbuilder, sounds more like a collection of tasks instead.